pkg/controllers/user/cis/utils.go

```diff
@@ -107,3 +110,68 @@ func ValidateClusterBeforeLaunchingScan(cluster *v3.Cluster) error {
 	}
 	return nil
 }
+
```

Comment:
since this is a util method, would you mind adding a comment on the top explain this function?

Reply:
Sure.